CreatureMethods.h

```diff
@@ -73,10 +90,10 @@ namespace LuaCreature
      * @param bool checkFaction = true : if `true`, the [Creature] must be the same faction as `friend` to assist
      * @return bool canAssist
      */
-    int CanAssistTo(lua_State* L, Creature* creature)
+    int CanAssistTo(lua_State *L, Creature *creature)
     {
-        Unit* u = Eluna::CHECKOBJ<Unit>(L, 2);
-        Unit* enemy = Eluna::CHECKOBJ<Unit>(L, 3);
+        Unit *u = Eluna::CHECKOBJ<Unit>(L, 2);
+        Unit *enemy = Eluna::CHECKOBJ<Unit>(L, 3);
```

Comment:
For example these

Reply:
Fixed them, was due to automatic code formatting